From 87bbb65c3c5fb7d248871d9c3bcbd3d27c781449 Mon Sep 17 00:00:00 2001
From: Venture Research <tech@ventureresearch.com>
Date: Wed, 6 Feb 2013 20:51:02 -0600
Subject: [PATCH] hack to force use of libc malloc

Hack to force libc usage as it seems the option to pass it in has been
removed in favor of magic.

Note that this of course doesn't allow tcmalloc and jemalloc, however
jemalloc wasn't building correctly.

Signed-off-by: Venture Research <tech@ventureresearch.com>

Update to work with 4.0.8
Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

Update to work with 8.0.0
Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

Update to work with 8.1.0
Signed-off-by: Randy MacLeod <Randy.MacLeod@windriver.com>

Upstream-Status: Inappropriate [Embedded specific]

---
 src/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 1437f41bd..5919f01e9 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -13,7 +13,8 @@
 # Just use 'make dep', but this is only needed by developers.
 
 release_hdr := $(shell sh -c './mkreleasehdr.sh')
-uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+# use fake uname option to force use of generic libc
+uname_S := "USE_LIBC_MALLOC"
 uname_M := $(shell sh -c 'uname -m 2>/dev/null || echo not')
 CLANG := $(findstring clang,$(shell sh -c '$(CC) --version | head -1'))
 
